documentation/reference.md

```diff
@@ -274,7 +274,7 @@ If you have additional data to report that does not fit any of the columns provi
 |Grant Programme:Title|The title of this grant programme.|string|False|
 |Grant Programme:Description|A description of this grant programme.|string|False|
 |Grant Programme:URL|A web link to more details of this grant programme.|uri|False|
-|Grant Programme:Last modified|-|date-time|False|
+|Grant Programme:Last modified|The date when information on this grant was last updated.|date-time|False|
```

Comment:
This is not about a grant as such, it is about the Grant Programme - this can be altered without affecting anything else